src/main/java/com/google/ehub/servlets/UserProfile.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.servlets;
+
+public class UserProfile {
```

Comment:
Is this a duplicate class? Looks like there is also `src/main/java/com/google/ehub/data/UserProfile.java`

We can probably remove this one.